docs/reference/mapping/dynamic/templates.asciidoc

```diff
@@ -14,6 +14,10 @@ name
 * <<path-match-unmatch,`path_match` and `path_unmatch`>> operate on the full
 dotted path to the field
 
+* if a dynamic template does not define `match_mapping_type` nor `match` nor
+`path_match`, then it won't match any field except fields are used with 
```

Comment:
I would slightly rephrase it to "then it won't match any field, but it can be referred to by name in `dynamic_template_hints` as part of a bulk request"

Reply:
++, I reworded it in https://github.com/elastic/elasticsearch/pull/69948/commits/d5a92310d866a74b032976f00cd6782608f3528a